index.html

```diff
@@ -1,13 +1,31 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <meta charset='utf-8'>
     <meta http-equiv='X-UA-Compatible' content='IE=edge'>
     <title>Awesome Books</title>
     <meta name='viewport' content='width=device-width, initial-scale=1'>
-    <link rel='stylesheet' type='text/css' media='screen' href='css/style.ccs'>
+    <link rel='stylesheet' type='text/css' media='screen' href='css/style.css'>
     <script src='js/main.js'></script>
```

Comment:
- [ ]  OPTIONAL :  strongly recommend you to import JS files only at the and of the document because your app may crush as JS is looking for nodes before they load. For example in this app you are looking for `"books-list"` id, when the script is loaded at the beginning, it may execute before the HTML loads and crush because it is not able to find this id in the document.  Secondly, consider always indicating path  with `./` notation: `<script src='./js/main.js' </script>`. Because withoud dot notation on some browsers on devices app may crush and JS will not load.